### ISSUE TITLE
localization via subdomain for language specific wiki pages

### ISSUE THREAD

**Issue by jywarren** [publiclab/plots2#90]
http://stackoverflow.com/questions/18770602/how-to-detect-the-subdomain-from-request-in-a-gem-file

http://es.publiclab.org would be nice!

I think we should be able to access this directly via the `request` object: `request.subdomain`

**Comment by ananyo2012** [comment]
@jiteshjha Is this something that you completed during your Internationalization project?

**Comment by jywarren** [comment]
I believe this was left unresolved, and has a related PR at #720, but I
don't know if there is starting code there. If Jitesh could link to
available guides/resources about using subdomain in this way, that would be
great.

I also believe there (maybe in #720) was a possible conflict with another
use of subdomains (for regions like nyc.publiclab.org) articulated in #720
but we could reconcile both uses, or have one take priority over the other.

**Comment by gauravano** [comment]
@jywarren we can try to implement this not now but maybe after resolving all the issues of Translation project. Also, here's a some guidance for this - https://guides.rubyonrails.org/i18n.html#setting-the-locale-from-the-domain-name

Also, here's warning about the approach we currently use - 
![our_approach](https://user-images.githubusercontent.com/20878070/54229440-43cd9400-452a-11e9-9b06-d2f3da6f499b.png)

:see_no_evil: 

Thanks!

**Comment by jywarren** [comment]
Yes, i think that's a good idea. And whoa, ok. We'll have to fix that!
